src/components/ButtonWithDropdownMenu.js

```diff
@@ -97,20 +103,31 @@ function ButtonWithDropdownMenu(props) {
                         shouldRemoveRightBorderRadius
                         style={[styles.flex1, styles.pr0]}
                         pressOnEnter
+                        large={props.buttonsizelarge}
```

Comment:
can we expose single param fetching constants like `LARGE`, `MEDIUM`? like `AVATAR_SIZE`. I am aware`Button` component exposes each option as a separate param, but it is not good pattern. Atleast we can avoid the same at ButtonWithDropdownMenu.